winter/pagination/page_position_argument_resolver.py

```diff
@@ -21,18 +21,18 @@ def resolve_argument(self, argument: ControllerMethodArgument, http_request: DRF
         raw_limit = http_request.query_params.get(self.limit_parameter_name)
         raw_offset = http_request.query_params.get(self.offset_parameter_name)
 
-        limit = self._parse_param(raw_limit, 'limit')
-        offset = self._parse_param(raw_offset, 'offset')
+        limit = self._parse_param(raw_limit, 'limit', False)
+        offset = self._parse_param(raw_offset, 'offset', True)
```

Comment:
Please add a keyword for this argument, so that it can be understood without reading `_parse_param` implementation

Reply:
Also one more suggestion:
```
limit = self._parse_int_param(raw_limit, 'limit', min_value=1)
offset = self._parse_int_param(raw_offset, 'offset', min_value=0)
```